engine/rest-client/src/main/java/org/datacleaner/restclient/ComponentController.java

```diff
@@ -75,7 +75,7 @@ public String createComponent(final String tenant, final String name, final Stri
      * @return
      */
     public ProcessOutput processComponent(final String tenant, final String id, final ProcessInput processInput)
-            throws ComponentNotFoundException;
+            throws RuntimeException;
```

Comment:
Not really a problem, but keeping ```throws RuntimeException``` instead of just deleting the throws declaration seems a bit unnecessary: ```throws ComponentNotFoundException``` didn't have any real effect, but it could help for documentation purposes. ```throws RuntimeException``` doesn't have any effect, and doesn't document anything.